src/handbook/development/releases/planning.md

```diff
@@ -4,70 +4,55 @@ navTitle: Planning
 
 # Planning
 
-Instructions for creating a release are [here](./process.md).
+This page provides a comprehensive overview of the planning processes and procedures that guide our product development at FlowFuse. As a continuously evolving software, FlowFuse employs a structured planning method to ensure timely releases, both for the FlowFuse Cloud and for our self-hosting users.
 
-## Cadence
+Instructions for creating a NPM package release are [here](./process.md).
 
-FlowFuse is released every four weeks, on a Thursday. We track releases using
-GitHub Milestones. The current schedule of releases can be seen [here](https://github.com/FlowFuse/flowfuse/milestones).
+Our planning process is continuously evolving as we find the best way to accommodate both a growing team and an expanding set of requirements for how and what we deliver.
 
-Our planning process is continually evolving as we find the right way to accommodate
-both a growing team and a growing set of requirements on both how and what we deliver.
+## Cadence
 
-The following reflects both where we are at today and where we want to get to.
+FlowFuse is continuously released to FlowFuse Cloud, and every four weeks, on a Thursday, it is packaged for users who are self-hosting FlowFuse. 
 
-We use GitHub issues for planning the work in a release. A Milestone should exist
-for the current release (N) and the next one (N+1). We do not schedule work beyond
-that as priorities can change through a release.
+### Milestones
 
----
+We track packages using GitHub Milestones. We utilize GitHub issues for planning work. Issues are continuously scheduled by the PM. Releases are structured following the [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
-### Week 2 - UX Planning
+Milestones indicate the week of the release instead of a Patch version. This leads to a specific structure for Milestones: **MAJORversion.MINORversion.WEEK**. For example, 1.14.2 would represent the second week of the release of 1.14. This method ensures that Milestones can be used as deadlines while allowing continuous planning.
```

Comment:
As mentioned when we discussed, I'm not 100% on this bit. Given a prioritised backlog, the expectation is everyone is working on the next most important task assigned to them and they get delivered in that order when they are ready.

It's perhaps as simple as calling it a 'target' not a 'deadline' - with an expectation that, in agile, things happen and targets move. 'deadline' feels too final.

Reply:
Yes, I'll change it to target.

Reply:
```suggestion
Milestones indicate the week of the release instead of a Patch version. This leads to a specific structure for Milestones: **MAJORversion.MINORversion.WEEK**. For example, 1.14.2 would represent the second week of the release of 1.14. This method ensures that Milestones can be used as targets while allowing continuous planning.
```